### PR TITLE
[zh] Optimize article translation and format

### PR DESCRIPTION
**Pull request by Adityayxt**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/ops/ambient/architecture/index.md [hunk 1]
```diff
@@ -8,80 +8,75 @@ test: n/a
...
+对与直接发送到 `Pod` 的，请求将在应用策略后才会被直接转发。
 
-对于到 `Service` 的请求，waypoint 还将应用路由和负载均衡。
-`Service` 默认只是路由到本身，在其端点之间进行负载均衡。
+对于发送到 `Service` 的请求，waypoint 还将应用路由和负载均衡。
+默认情况下，`Service` 会简单地将请求路由到本身，在其端点之间进行负载均衡。
 这可以重载为针对 `Service` 的路由。
 
-例如，以下策略将确保到 `echo` 服务的请求被转发到 `echo-v1`：
+例如，以下策略将确保到 `echo` 服务的请求转发到 `echo-v1`：
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/architecture/index.md:79]:
```suggestion
例如，以下策略将确保到 `echo` 服务的请求被转发到 `echo-v1`：
```
被  表示无需人工干预的系统行为，不要改为“错”

**Review by windsonsea — COMMENTED** [review]
建议尽量先读懂原文，再改动

**Comment by Adityayxt** [comment]
> 建议尽量先读懂原文，再改动

已经读过原文，并且找到相关内容的资料：https://istio.io/latest/blog/2023/rust-based-ztunnel/，更合理的翻译

File: content/zh/docs/ops/ambient/architecture/index.md [hunk 2]
```diff
@@ -8,80 +8,75 @@ test: n/a
...
 
-#### 出站{#outbound}
+#### 出站 {#outbound}
 
-当捕获的 Pod 发出出站请求时，它将被透明地重定向到 ztunnel 来决定如何转发请求以及转发到哪儿。
-总之，流量路由行为就像 Kubernetes 默认的流量路由；
-到 `Service` 的请求将被发送到 `Service` 内到一个端点，
-而直接到 `Pod` IP 的请求将直接转到该 IP。
+当被捕获的 Pod 发出出站请求时，它将被透明地重定向到 Ztunnel，Ztunnel 来决定如何转发请求以及转发到哪儿。
+一般情况下，流量路由行为就像 Kubernetes 默认的流量路由一样；
```
[the remark this answers is not in the document]
Comment (Adityayxt) [reply content/zh/docs/ops/ambient/architecture/index.md:34]: In general, the traffic routing behaves just like Kubernetes default traffic routing; 原文如上，认为用 一般情况下， 更准确

**Review by windsonsea — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**